draft-ietf-quic-load-balancers.md

```diff
@@ -852,9 +873,8 @@ the server can issue an up-to-date token to the client.
 
 When issuing Retry or NEW_TOKEN tokens, the server MUST include the client IP
 address in the authenticated data as specified in
-{{token-protection-with-aead}}. It MUST include a means of distinguishing
-service-generated Retry tokens, server-generated Retry tokens (if different),
-and NEW_TOKEN tokens.
+{{token-protection-with-aead}}. The ODCIL and RSCIL fields are zero for
+NEW_TOKEN tokens, making them easily distinguishable from Retry tokens.
 
```

Comment:
The text about port verification should probably be added there, or in the next paragraphs.